spec/order_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'order'
+
+describe Order do
+  it {is_expected.to respond_to(:order)}
```

Comment:
This test is vacuous when you consider that you are testing the same method on line 15...